.gitignore

```diff
@@ -1,4 +1,4 @@
 .env
-.anonymized_text.txt
-.__pycache__/
+anonymized_text.txt
+__pycache__
```

Comment:
those changes are already on `main` branch. please do a git rebase on main to avoid this. here are the steps:

```bash
git checkout main
git pull origin main --rebase
git checkout feat/my-branch
git rebase main
git push origin feat/my-branch --force
```